TestLoader.cs

```diff
@@ -0,0 +1,60 @@
+﻿using System.Collections.Generic;
+using System.IO.Compression;
+using System.IO;
+using System.Text.Json;
+using System.Text;
+using System.Threading.Tasks;
+using WasmBenchmarkResults;
+using System.Runtime.InteropServices.JavaScript;
+using System;
+
+Console.WriteLine("Hello World!");
```

Comment:
hello could go away ?

Reply:
If I try to remove it, I get this error when I try to build the app:  Program does not contain a static 'Main' method suitable for an entry point [C:\Users\t-spahontu\Desktop\PerformanceTool\PerformanceTool.csproj]

Reply:
You probably need to also remove the `App.MONO.mono_run_main`